docs/src/main/sphinx/connector/druid.rst

```diff
@@ -57,6 +57,73 @@ name from the properties file.
 Type mapping
 ------------
 
+Because Trino and Druid each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Druid type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Druid types to the corresponding Trino types according to the
+following table:
+
+.. list-table:: Druid type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
```

Comment:
Several of these entries are one-to-many relationships which doesn't make sense in the context of type mapping, so I expect we'll be removing or changing a number of these.

Reply:
Note that https://druid.apache.org/docs/latest/querying/sql-data-types.html talks about Druid's own SQL queries - not how Druid data types get exposed to other systems which is probably a source of common confusion.

Reply:
@hashhar should i remove that link?

Reply:
+1 to removing the sentence and link on line 100. If that document is confusing, and the newly-updated table is complete and correct, then I don't think it's necessary.

Reply:
Done